md/moves.py

```diff
@@ -56,3 +56,45 @@ def n_accepted(self):
     @property
     def n_proposed(self):
         return np.sum(self.n_proposed_by_move)
+
+
+from scipy.special import logsumexp
+
+
+class MultipleTryMetropolis:
+    def __init__(self, K):
```

Comment:
Would add reference to https://www.stat.purdue.edu/~fmliang/papers/2000/Mtry.pdf in docstring, and a comment that implicitly `lambda(x, y) = 1 / proposal_pdf(x, y)`

Reply:
willfix

Reply:
Based on discussion this morning, my suggestion that implicitly "lambda(x, y) = 1 / proposal_pdf(x, y)" was incorrect -- this is not implied by the current implementation of `w(x, y)`. Will re-visit this more carefully, to confirm that the `lambda(x, y)` implied by the implementation is symmetric!